neqo-transport/src/recovery.rs

```diff
@@ -38,12 +54,13 @@ pub enum RecoveryToken {
 #[derive(Debug, Clone)]
 pub struct SentPacket {
     ack_eliciting: bool,
-    //in_flight: bool, // TODO needed only for cc
-    //size: u64, // TODO needed only for cc
     time_sent: Instant,
     pub(crate) tokens: Vec<RecoveryToken>,
```

Comment:
Do we even need pub(crate) if this struct is only ever available to the crate.  Can't this be public?

Reply:
sure ok